config/locales/en/lookup.yml

```diff
@@ -36,15 +36,15 @@ en:
       guidance_label: Find out what the Tier %{level} rules are
       match: "We've matched the postcode"
       level_one:
-        heading: "This area is in Tier 1: Medium alert"
+        heading: "This area is in <div>Tier 1: Medium alert</div>"
         changing_alert_level: "At the moment %{area} is in Tier 1."
         guidance_link: "/guidance/tier-1-medium-alert"
       level_two:
-        heading: "This area is in Tier 2: High alert"
+        heading: "This area is in <div>Tier 2: High alert</div>"
         changing_alert_level: "At the moment %{area} is in Tier 2."
         guidance_link: "/guidance/tier-2-high-alert"
       level_three:
-        heading: "This area is in Tier 3: Very High alert"
+        heading: "This area is in <div>Tier 3: Very High alert</div>"
```

Comment:
Not sure that having a `div` inside a heading is valid HTML. Could this be a span with a class to prevent wrapping on the whitespace instead? (might be something in the DS that does this already).

Reply:
`white-space: nowrap`

Reply:
@andysellick - updated

Reply:
Great.